source/slides/03_linux_basics.rst

```diff
@@ -0,0 +1,443 @@
+.. _03_linux_basics:
+
+Linux Basics (Day 3)
+====================
+
+Filesystems, system booting and OS Installation
+
+Before we begin
+---------------
+
+.. code-block:: bash
+
+  $ sudo yum install man man-pages man-pages-overrides
```

Comment:
Ya, I missed that part while cleaning the slides up for the class. Thanks for pointing that out. FWIW I think our images don't include man by default.